internal/api/genapi/templates.go

```diff
@@ -671,8 +689,13 @@ var structTemplate = template.Must(template.New("").Funcs(
 package {{ .Package }}
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"errors"
 	"fmt"
+	"io"
+	"net/url"
```

Comment:
I added some imports that were already used but added but post-generation formatting steps.